holidays/countries/sint_maarten.py

```
@@ -0,0 +1,125 @@
+from datetime import date
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import NOV
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SUN_TO_NEXT_MON,
+    SAT_TO_PREV_FRI,
+    SUN_TO_NEXT_TUE,
+    SUN_TO_PREV_SAT,
+)
```

Pylint (3.3.7)

[error] No name 'SUN_TO_PREV_SAT' in module 'holidays.observed_holiday_base'

(E0611)